### PR TITLE
update notebook templates

### PR DESCRIPTION
**Pull request by ebattat**
Fixes:
1. Update working notebooks using latest benchmark-runner package
2. Fixing widgets changes by using getter for every change variable: start_datetime, end_datetime and elastic_index

### CODE REVIEW

File: benchmark_runner/jupyterlab/templates/elasticsearch_operations/elasticsearch_widgets.py [hunk 1]
```diff
@@ -65,4 +87,8 @@ def on_value_change(change):
         end_date_picker.observe(on_value_change, names='value')
         start_time_picker.observe(on_value_change, names='value')
         end_time_picker.observe(on_value_change, names='value')
-        return start_datetime, end_datetime
+
+        display(start_date_picker)
+        display(start_time_picker)
+        display(end_date_picker)
+        display(end_time_picker)
```
Comment (athiruma) [inline benchmark_runner/jupyterlab/templates/elasticsearch_operations/elasticsearch_widgets.py:94]: Missing New Line

Comment (athiruma) [inline benchmark_runner/jupyterlab/templates/elasticsearch_operations/elasticsearch_widgets.py:94]: Missing New Line

Comment (ebattat) [reply benchmark_runner/jupyterlab/templates/elasticsearch_operations/elasticsearch_widgets.py]: Fixed

File: benchmark_runner/jupyterlab/templates/elasticsearch_operations/elasticsearch_widgets.py [hunk 2]
```diff
@@ -41,16 +66,13 @@ def choose_datetime(self):
...
         end_time_picker = widgets.TimePicker(description='End Time', value=time(hour=self.DEFAULT_TIME, minute=self.DEFAULT_TIME, second=self.DEFAULT_TIME))
 
-        display(start_date_picker)
-        display(start_time_picker)
-        display(end_date_picker)
-        display(end_time_picker)
-
         def on_value_change(change):
             global start_datetime
             global end_datetime
```
Comment (athiruma) [inline benchmark_runner/jupyterlab/templates/elasticsearch_operations/elasticsearch_widgets.py:78]: can we declare these two variables in the __init__() method and access using getter methods?

Comment (ebattat) [reply benchmark_runner/jupyterlab/templates/elasticsearch_operations/elasticsearch_widgets.py:78]: @athiruma, Jupyter widgets are working quite different, global variable MUST declare inside on_value_change.
After tr

**Review by athiruma — APPROVED** [review]
LGTM

**Pull request merged**